parlai/utils/testing.py

```diff
@@ -53,6 +53,14 @@
 except ImportError:
     BPE_INSTALLED = False
 
+try:
+    import maskrcnn_benchmark  # noqa: F401
+    import cv2  # noqa: F401
+
+    DETECTRON_AVAILABLE = True
```

Comment:
Random thought - if these import statements break in the future, does that mean that our Detectron CI checks will stop silently because `skipUnlessDetectron()` will skip them?

Reply:
yes, though that could be said for every import statement in this file

Reply:
Yeah, I see that a ton of them have the same try/except logic